ui/components/app/modular-confirmations/confirmation-row.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import Tooltip from '../../ui/tooltip/tooltip';
+import { Box, Icon, IconName, Text } from '../../component-library';
+import {
+  AlignItems,
+  BackgroundColor,
+  BorderRadius,
+  Color,
+  Display,
+  FlexDirection,
+  FlexWrap,
+  JustifyContent,
+  TextColor,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+
+export type ConfirmationRowProps = {
+  label: string;
+  children: React.ReactNode | string;
+  tooltip?: string;
+  variant?: 'default' | 'error' | 'warning';
```

Comment:
[nit] we can store the variants in a const similar to our TextVariant or BUTTON_VARIANT. using these as the keys for below can help avoid typo errors

Reply:
Done!